exporters/fluentd/cmake/opentelemetry-cpp-fluentd-config.cmake.in

```diff
@@ -0,0 +1,66 @@
+#.rst:
+# opentelemetry-cpp-fluentd.config.cmake
+# --------
+#
+# Find the native opentelemetry-cpp-fluentd includes and library.
+#
+#
+# Result Variables
+# ^^^^^^^^^^^^^^^^
+#
+# This module defines the following variables:
+#
+# ::
+#
+#   OPENTELEMETRY_CPP_FLUENTD_INCLUDE_DIRS  - Include directories of opentelemetry-cpp-fluentd.
+#   OPENTELEMETRY_CPP_FLUENTD_LIBRARY_DIRS  - Link directories of opentelemetry-cpp-fluentd.
+#   OPENTELEMETRY_CPP_FLUENTD_LIBRARIES     - List of libraries when using opentelemetry-cpp-fluentd.
+#   OPENTELEMETRY_CPP_FLUENTD_FOUND         - True if opentelemetry-cpp-fluentd found.
+#   OPENTELEMETRY_CPP_FLUENTD_VERSION           - Version of opentelemetry-cpp-fluentd.
+#
+# ::
+#   opentelemetry-cpp-fluentd::trace                    - Imported target of opentelemetry-cpp-fluentd::trace
+#   opentelemetry-cpp::logs                             - Imported target of opentelemetry-cpp-fluentd::logs
+
+# =============================================================================
+# Copyright 2020 opentelemetry.
+#
+# Distributed under the Apache License (the "License"); see accompanying file
+# LICENSE for details.
+# =============================================================================
+set(OPENTELEMETRY_CPP_FLUENTD_VERSION
+    "@OPENTELEMETRY_CPP_FLUENTD_VERSION@"
+    CACHE STRING "opentelemetry-cpp-fluentd version" FORCE)
+
+@PACKAGE_INIT@
+
+message(OPENTELEMETRY_CPP_FLUENTD_VERSION)
+find_package(Threads)
+
+set_and_check(OPENTELEMETRY_CPP_FLUENTD_INCLUDE_DIRS "@PACKAGE_INCLUDE_INSTALL_DIR@")
+set_and_check(OPENTELEMETRY_CPP_FLUENTD_LIBRARY_DIRS "@PACKAGE_CMAKE_INSTALL_LIBDIR@")
+
+include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@-target.cmake")
+
+set(OPENTELEMETRY_CPP_FLUENTD_LIBRARIES)

```

Comment:
Is this needed?

Reply:
Good catch. The was supposed to be used in populating the available targets. Somehow missed out on the recent local changes in this PR. Have fixed it now.